src/components/DataProvider/index.js

```diff
@@ -315,6 +325,7 @@ DataProvider.propTypes = {
   children: PropTypes.node.isRequired,
   defaultLoader: PropTypes.func,
   series: seriesPropType.isRequired,
+  onSubDomainChanged: PropTypes.func,
```

Comment:
Could you add a `// (args) => null` comment? I don't know what this callback gets back.

Reply:
done